apps/rush-lib/src/api/RushGlobalFolders.ts

```diff
@@ -0,0 +1,39 @@
+// Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT license.
+// See LICENSE in the project root for license information.
+
+import * as path from 'path';
+import { Utilities } from '../utilities/Utilities';
+
+/**
+ * This class provides global folders that are used for rush's internal install locations.
+ *
+ * @internal
+ */
+export class RushGlobalFolders {
```

Comment:
>RushGlobalFolders [](start = 13, length = 17)

Class names should never be plural.

Reply:
RushGlobalStorage?

---
In reply to: [231351163](https://github.com/Microsoft/web-build-tools/pull/934#discussion_r231351163) [](ancestors = 231351163)